src/issues/currentIssue.ts

```diff
@@ -130,19 +129,30 @@ export class CurrentIssue extends Disposable {
 		return undefined;
 	}
 
-	private async createOrCheckoutBranch(branch: string): Promise<boolean> {
+	private async createOrCheckoutBranch(branch: string, silent: boolean): Promise<boolean> {
+		let localBranchName = branch;
 		try {
-			if (await this.getBranch(branch)) {
-				await this.manager.repository.checkout(branch);
+			const isRemoteBranch = branch.startsWith('origin/');
```

Comment:
A remote branch doesn't have to start with `origin`, `origin` is just used as the default remote name by convention.